content/en/docs/components/pipelines/user-guides/core-functions/run-a-pipeline.md

```diff
@@ -1,83 +1,150 @@
+        class="mt-3 mb-3 border border-info rounded">
 
-<img src="/docs/images/pipelines/submit-a-pipeline-on-dashboard.png" 
-  alt="Upload pipeline button"
-  class="mt-3 mb-3 border border-info rounded">
+3. Upload the pipeline `.yaml`, `.zip` or `.tar.gz` file, populate the upload form, then click `Create`.
 
-3. Upload the pipeline IR YAML file or an archived pipeline as a .zip or .tar.gz file, populate the upload pipeline form, and click “Create”.
+   <img src="/docs/images/pipelines/upload-a-pipeline.png" 
+        alt="Upload pipeline screen" 
+        class="mt-3 mb-3 border border-info rounded">
 
-<img src="/docs/images/pipelines/upload-a-pipeline.png" 
-  alt="Upload pipeline screen"
-  class="mt-3 mb-3 border border-info rounded">
+4. From the "Runs" tab, select `+ Create run`:
 
-4. From the Runs tab, select "+ Create run":
+   <img src="/docs/images/pipelines/create-run.png" 
+        alt="Create run button"
+        class="mt-3 mb-3 border border-info rounded">
 
-<img src="/docs/images/pipelines/create-run.png" 
-  alt="Create run button"
-  class="mt-3 mb-3 border border-info rounded">
+5. Select the pipeline you want to run, populate the run form, then click `Start`:
 
-5. Choose the pipeline you uploaded, provide a name, any run parameters, and click "Start".
-<img src="/docs/images/pipelines/start-a-run.png" 
-  alt="Start a run screen"
-  class="mt-3 mb-3 border border-info rounded">
+   <img src="/docs/images/pipelines/start-a-run.png" 
+        alt="Start a run screen"
+        class="mt-3 mb-3 border border-info rounded">
 
+## Run Pipeline - KFP SDK Client
 
-## 2. Run from the KFP SDK client
-You may also programatically submit pipeline runs from the KFP SDK client. The client supports two ways of submitting runs: from IR YAML or from a Python pipeline function. For either approach, start by instantiating a `Client` using the `host` URL of your KFP instance:
+You may also programmatically submit pipeline runs from the KFP SDK client. 
+The client supports two ways of submitting runs: _from IR YAML_ or _from a Python pipeline function_. 
+
+{{% alert title="Note" color="dark" %}}
+See the [Connect the SDK to the API](/docs/components/pipelines/user-guides/core-functions/connect-api/) guide for more information about creating a KFP client.
+{{% /alert %}}
+
+For either approach, start by instantiating a [`kfp.Client()`][kfp-client]:
 
 ```python
-from kfp.client import Client
-client = Client(host='<YOUR_HOST_URL>')
+import kfp
+
+# TIP: you may need to authenticate with the KFP instance
+kfp_client = kfp.Client()
 ```
 
-To submit IR YAML for execution use the `.create_run_from_pipeline_package` method:
+To submit __IR YAML__ for execution use the [`.create_run_from_pipeline_package()`][kfp-client-create_run_from_pipeline_package] method:
 
 ```python
-client.create_run_from_pipeline_package('pipeline.yaml', arguments={'param': 'a', 'other_param': 2})
+#from kfp import compiler, dsl
```

Comment:
Would you like to keep the code commented?

Reply:
Yes, the idea is to highlight that there are two stages to this:

1. Create the YAML
2. Submit the YAML

I also dont want people accidentially overwiting files by bildly runing the compile code.